docs/build_steps.rst

```diff
@@ -215,6 +221,36 @@ Alpine Commands
    setup:
    - !Alpine v3.4
 
+.. step:: AlpineRepo
+
+   Adds arbitrary alpine repository. For example to add testing repository::
+
+     - !AlpineRepo
+       url: http://nl.alpinelinux.org/alpine/
+       branch: edge
+       repo: testing
+       alias: testing
+     - !Install [app@testing]
+
+   Options:
+
+   url
+     Url to the repository. Default is the mirror url from the current alpine
+     distribution.
+
+   branch
+     Branch of the repository. For example ``v3.4``, ``edge``. Default is
+     the version of the current alpine distribution.
+
+   repo
+     Repository to fetch packages from. For example ``main``, ``community``,
+     ``testing``. **Required**.
+
+   alias
+     Alias or "pin" for this repository. Alpine package manager will now
+     by default only use the untagged repositories. Adding an alias to
```

Comment:
"Alias or pin" and later "untagged repositories", it's unclear what "untagged" means